utils/eventtracking.js

```diff
@@ -48,19 +50,20 @@ const track = async (eventType, eventProperties, options) => {
             return;
         }
 
-        if (shouldTrackID(shellSettings)) {
+        if (options.accountId && shouldTrackID(shellSettings)) {
```

Comment:
@itegulov is this change intended?
I think users can avoid tracking even if they do not pass any `accountId` in command.